### ISSUE TITLE
Kernel for Diting (12Tpro/K50Ultra)?

### ISSUE THREAD

**Issue by trinhloivn** [Xiaomi-Diting-Kernel-Dump/android_kernel_xiaomi_diting#1]
Is this true?what's going on?

**Comment by Laulan56** [comment]
> Is this true?what's going on?

Yes, as you can see, Xiaomi did not release the kernel source code for diting. But they have open sourced a good portion of the kernel drivers used by diting, so I was able to dump the kernel source code for diting. However, I don't have diting as a device. So I can't test the availability of this kernel. I got the source code through my existing kernel development experience, and a good part of it is my reverse. I am releasing them in the hope that they will be improved by the community.

More information: https://github.com/Xiaomi-Diting-Kernel-Dump/manifest
